### ISSUE TITLE
Scripting Error #8

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#538]
![image](https://github.com/EmeraldLoc/tag/assets/168046906/7e1c9459-1eb1-496e-bb16-ce069314c8ca)

**Comment by EmeraldLoc** [comment]
How did you get this error?

**Comment by TheMan6900** [comment]
simply by joinning the game 💀

**Comment by Bear64DX** [comment]
On CoopDX

**Comment by TheMan6900** [comment]
> On CoopDX

oh...

**Comment by EmeraldLoc** [comment]
> On CoopDX

Coopdx is the only version that supports custom themes, due to the keys required being absurdly large.

**Comment by Bear64DX** [comment]
Yeah. The bug is still kinda annoying, since it makes it hard to tell if the code I wrote has a problem (I mean console can but... thats not the point)
